pkgs/by-name/kc/kcl/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "kcl";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-ZjEMgQukhBGY3LWqsGmLj3lKfLtNNaQugQs0cSLMb80=";
+  };
+
+  vendorHash = "sha256-jmqKMB85HxAlwH7FVjHrLCZQYuAJrguRfzIz1yMypjw=";
+
+  ldflags = [
+    "-X=kcl-lang.io/cli/pkg/version.version=${version}"
+  ];
+
+  subPackages = [ "cmd/kcl" ];
+
+  meta = with lib; {
+    description = "The KCL Command Line Interface (CLI)";
```

```suggestion
    description = "A command line interface for KCL programming language";
```

This is excerpted from the repo description, but this description is not good enough, it almost duplicates the name of the package.

---

fixed. PTAL